src/main/resources/config/application-dev.yml

```diff
@@ -28,21 +28,6 @@ spring:
           - git-branch=${git.branch:}
       host: localhost
       port: 8500
-  datasource:
-    type: com.zaxxer.hikari.HikariDataSource
-    url: jdbc:h2:mem:mstemplate;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
-    username: mstemplate
-    password:
-    hikari:
-      poolName: Hikari
-      auto-commit: false
-  h2:
-    console:
-      # disable spring boot built-in h2-console since we start it manually with correct configuration
-      enabled: false
-  jpa:
-    database-platform: tech.jhipster.domain.util.FixedH2Dialect
-    database: H2
```

Comment:
H2 it`s MAIN for local testing, add NEW profile like PG dev, if you want (and check first, it can be already exists)